proprietary/Components/src/utrecht/form-field.tokens.json

```diff
@@ -0,0 +1,21 @@
+{
+  "utrecht": {
+    "form-field": {
+      "margin-block-start": { "value": "{denhaag.space.block.lg}" },
+      "margin-block-end": { "value": "{denhaag.space.block.lg}" },
+      "label": {
+        "margin-block-end": { "value": "{denhaag.space.block.2xs}" }
+      },
+      "description": {
+        "color": { "value": "{denhaag.color.grey.4}" },
+        "font-family": { "value": "{utrecht.document.font-family}" },
+        "font-size": { "value": "{denhaag.typography.scale.s.font-size}" },
+        "font-weight": { "value": "{denhaag.typography.weight.regular}" },
+        "line-height": { "value": "{denhaag.typography.scale.s.line-height}" },
+        "error": {
+          "color": { "value": "{denhaag.color.red.3}" }
+        }
```

Comment:
These tokens belong to the [`utrecht.form-field-description` tokens](https://nl-design-system.github.io/utrecht/storybook/?path=/docs/css-form-field-description-design-tokens--page). In that component `error` is called `invalid`, after `:invalid`.

Reply:
I'll check if I can remove these. (Assuming that's what you meant.)